internal/provider/data_source_pullzone_test.go

```diff
@@ -0,0 +1,36 @@
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourcePullZone(t *testing.T) {
+	resourceName := randResourceName()
+	resource.UnitTest(t, resource.TestCase{
```

Comment:
```suggestion
	resource.Test(t, resource.TestCase{
```

It creates a pull-zone on bunny, so it should be an acceptance test